### ISSUE TITLE
Pointer areas

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/expression-exchange#100]
Very few are overridden. Has the sim been vetted for what touch areas should be expanded?

Code review from https://github.com/phetsims/expression-exchange/issues/88

**Comment by jbphet** [comment]
I made some minor adjustments, and tested on iPad 2, and it all feels pretty usable to me.  Assigning to @phet-steele for verification on upcoming dev release.

**Comment by amanda-phet** [comment]
1 and -1 look funny on the Negatives screen. 

Also, I know that with a toggle you can generally click the images next to the toggle instead of dragging the actual toggle knob. I don't see the touch areas outlined with the query parameter but I can check on an iPad to see that they are functioning ok.

**Comment by amanda-phet** [comment]
Confirmed with iPad that the touch areas for the toggle are not large enough (or existent?). Can we make sure that the images for the coin and x are comfortably large for touch?

Currently, only the toggle knob has a larger touch area: 
![screen shot 2017-06-22 at 1 53 31 pm](https://user-images.githubusercontent.com/7852916/27453067-590334e6-5752-11e7-99c8-b7384b8f2274.png)

**Comment by jbphet** [comment]
As for 1 and -1, I think @amanda-phet was seeing the pointer areas for items that were clipped from the carousel.  I went ahead and added pointer areas to the constant coin terms though, just to avoid confusion, and it might make them more consistent to interact with.

As for the AB switch, I've expanded the touch areas for the nodes used and shifted these touch areas so that they don't overlap with that of the switch, here's a screenshot with `showPointerAreas` turned on.

![exex-touch](https://user-images.githubusercontent.com/3082831/27460573-e9ff7e72-5771-11e7-82b7-ec71f5d754ed.PNG)

Back to you @amanda-phet.
